gun_scraper/notifier.py

```diff
@@ -38,9 +38,10 @@ def send_email_notification(guns_list: List[Dict]):
         email_config["smtp_server"], email_config["ssl_port"], context=context
     ) as server:
         server.login(email_config["username"], email_config["password"])
-        server.sendmail(
-            email_config["sender"],
-            email_config["receiver"],
-            message.as_string(),
-        )
+        # server.sendmail(
+        #     email_config["sender"],
+        #     email_config["receiver"],
+        #     message.as_string(),
+        # )
+        logger.warning(f"Email intended to be sent: {message.as_string()}")
```

Comment:
Don't forget to revert these changes
```suggestion
        server.sendmail(
            email_config["sender"],
            email_config["receiver"],
            message.as_string(),
        )
```